tasks/ubuntu/packages.yml

```diff
@@ -28,83 +28,41 @@
   apt: update_cache=yes
 
 - name: Install required system packages
-  apt: pkg={{ item }} state={{ package_state }} install_recommends=no
-  with_items:
-    - ant
-    - cmake
-    - curl
-    - g++
-    - gcc
-    - gfortran
-    - git-core
-    - libffi-dev
-    - liblapack-dev
-    - libncurses5-dev
-    - libopenblas-dev
-    - libpam0g-dev
-    - libpq-dev
-    - libsparsehash-dev
-    - make
-    - mercurial
-    - nginx-extras
-    - openssh-server
-    - patch
-    - pkg-config
-    - python-boto
-    - python-dev
-    - python-prettytable
-    - python-psycopg2
-    - python-virtualenv
-    - python-pip
-    - rsync
-    - samtools
-    - slurm-drmaa-dev
-    - supervisor
-    - swig
-    - sysstat
-    - unzip
-    - wget
-    - zlib1g-dev
+  apt:
+    pkg: ['ant', 'cmake', 'curl', 'g++', 'gcc', 'gfortran', 'git-core',
```

Comment:
Is there a reason to switch the list syntax ?
```
apt:
  pkg:
    - ant
    - cmake
    ...
```
should work too. This looks cleaner to me, and you'd get cleaner diffs when updating.

Reply:
Well, I read the warnings of ansible 2.7 too fast. I also prefer the dashes for list description. I should definitely work